arch/arm/src/armv8-m/arm_lazyexception.S

```diff
@@ -210,6 +210,7 @@ exception_common:
 	 */
 
 	setintstack	r2, r3
+	sub		sp, #8					/* Adjust SP (minus 8-byte offset) */
```

Comment:
why sub 8?

Reply:
The same reason as what I commented on arch/arm/src/armv7-m/gnu/arm_lazyexception.S

Reply:
OK, I will remove the line.

Reply:
Done